styx-scheduler-service/src/main/java/com/spotify/styx/api/SchedulerResource.java

```diff
@@ -252,21 +253,31 @@ private StatusType eventInjectorHelper(Event event) {
       return Response.forStatus(BAD_REQUEST.withReasonPhrase(e.getMessage()));
     }
 
-    // Verifying active
```

Comment:
Why is this verification removed?

Reply:
The `triggerListener.event()` returned future we wait for (a few lines after this) will throw if the instance is already active